bitmap-steven-hawa-caleb/bitmap-transformer.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const cliInterpreter = require('./lib/cli-interpreter.js');
+const invert = require('./lib/invert-colors.js');
+const grayscale = require('./lib/grayscale-colors.js');
+const bluescale = require('./lib/bluescale-colors.js');
+const psychedelic = require('./lib/psychedelic.js');
```

Comment:
@maschigokae @BatemanVO @abdih17 `require` statements look good, along with the separation of your transform helpers by containing them in a `lib` directory